catbus-navigator/web-ui
# CN-3-Display-User-Location-On-Map

# Description

This pull request adds the functionality to display the user's location on a Mapbox map in the Map component. The feature was implemented per the user story requirement, which stated that the user's location should be displayed on the map.

Fixes # [CN-3](https://jpt4.atlassian.net/browse/CN-3)

# How Has This Been Tested?

To test the user location display feature on the Mapbox map, you can follow these steps:

1. Load the application that contains the Map component.
2. Ensure that the application has permission to access your location. This can be done by checking the browser settings or allowing the application to access your location when prompted.
3. Navigate to the map view in the application.
4. Click on the "locate me" button that should be visible on the map.
5. The map should now center on your current location and display a marker indicating your location on the map.
6. You can move around and see that the marker updates its position as you move.

If the above steps are successful, the feature works as expected, and the user's location will be displayed on the Mapbox map.

# Checklist:

- [x] Have you merged the main into this branch and resolved any conflicts?
- [x] Have you been able to update any necessary documentation?



[CN-3]: https://jpt4.atlassian.net/browse/CN-3?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
